python-client/giskard/core/suite.py

```diff
@@ -20,6 +23,41 @@
 suite_input_types: List[type] = [Dataset, BaseModel, str, bool, int, float, SlicingFunction, SlicingFunction]
 
 
+def parse_function_arguments(client, project_key, function_inputs):
```

Comment:
Could you import this from `giskard/ml_worker/server/ml_worker_service.py` instead of copying it? or centralise it a `utils` somewhere?

Reply:
The issue is on `ml_worker_service` we have a proto file while on suite a `json` (dict) file so we don't have the same structure and methods